src/schema/posts.ts

```diff
@@ -2215,6 +2225,39 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       }
       return { _: true };
     },
+    clickbaitPost: async (_, { id }: { id: string }, ctx: AuthContext) => {
+      const { contentQuality }: Pick<Post, 'contentQuality'> = await ctx.con
+        .getRepository(Post)
+        .findOneOrFail({
+          where: { id },
+          select: ['contentQuality'],
+        });
+
+      const is_clickbait_probability = parseFloat(
+        (contentQuality.is_clickbait_probability as unknown as string) || '0.0',
+      );
+      const clickbaitTitleProbabilityThreshold =
+        remoteConfig.vars.clickbaitTitleProbabilityThreshold || 1.0;
+
+      // We +/- 10 as it is a number that is above the natural range from Yggdrasil,
+      // so it will be easy to identify if the value was changed by a user.
```

Comment:
Do we need values outside of `0..1` now that we have `manual_clickbait_probability`?

Reply:
I guess we could check for either manual_clickbait_probability or is_clickbait_probability, and just leave is_clickbait_probability alone

Reply:
To set it to be either above or below threshold

Reply:
I don't understand, we can just set to `1` if moderator sets as clickbait?

Reply:
Yeah, but should also be able to set as not clickbait/ revert to original. 
So if we just keep the `manual_clickbait_probability` for post updated, we don't need to add logic to title field to care about the `manual_clickbait_probability` flag.

Reply:
Could `manual_clickbait_probability` then also just become 0-1 and if its set we use it, if not we use `clickbait_probability` from yggdrasil? Then reverting is just a matter of clearing that one field and then we use whatever yggdrasil set the last time?

Reply:
Refactored it so that it prefers `manual_clickbait_probability` when set. And mutation will remove flag if it is set, so to revert to original value.